manifests/init.pp

```diff
@@ -115,104 +139,59 @@
   Boolean $candlepin_db_ssl = $::katello::params::candlepin_db_ssl,
   Boolean $candlepin_db_ssl_verify = $::katello::params::candlepin_db_ssl_verify,
   Boolean $candlepin_manage_db = $::katello::params::candlepin_manage_db,
+
+  Boolean $enable_candlepin = $::katello::params::enable_candlepin,
+  Boolean $enable_qpid = $::katello::params::enable_qpid,
+  Boolean $enable_qpid_client = $::katello::params::enable_qpid_client,
+  Boolean $enable_pulp = $::katello::params::enable_pulp,
+  Boolean $enable_application = $::katello::params::enable_application,
+
+  String $candlepin_hostname = $::katello::params::candlepin_hostname,
+  String $pulp_hostname = $::katello::params::pulp_hostname,
+  String $qpid_hostname = $::katello::params::qpid_hostname,
+  Stdlib::HTTPUrl $candlepin_url = "https://${candlepin_hostname}:8443/candlepin",
```

Comment:
Would it be a problem if we only exposed the hostname variables and not the URLs for now?